imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -213,25 +248,32 @@ def update(self) -> None:
         # update every stress period.
 
         # exchange stages from Ribasim to MODFLOW
-        self.exchange_mod2rib()
+        if self.has_ribasim:
+            self.exchange_mod2rib()
 
         # Do one MODFLOW 6 - MetaSWAP timestep
-        self.update_modflow6_metaswap()
-
-        # exchange drainage fluxes from MODFLOW 6 to Ribasim
-        self.exchange_rib2mod()
 
-        # Update Ribasim until current time of MODFLOW 6
-        self.ribasim.update_until(self.get_current_time() * days_to_seconds(self.delt))
+        if self.has_metaswap:
+            self.update_MODFLOW6_MetaSWAP()
+        else:
+            self.update_MODFLOW6()
+
+        if self.has_ribasim:
+            # exchange drainage fluxes from MODFLOW 6 to Ribasim
+            self.exchange_rib2mod()
+            # Update Ribasim until current time of MODFLOW 6
+            self.ribasim.update_until(
+                self.get_current_time() * days_to_seconds(self.delt)
+            )
 
-    def update_modflow6_metaswap(self) -> None:
+    def update_MODFLOW6_MetaSWAP(self) -> None:
```

Comment:
Please move it back to lower cases. I changed it based on a comment of Julian, I think you inherited this from my original branch.